curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/650c6082e5586f9e3acfcd3b.md

```diff
@@ -0,0 +1,72 @@
+---
+id: 650c6082e5586f9e3acfcd3b
+title: Step 52
+challengeType: 20
+dashedName: step-52
+---
+
+# --description--
+
+Now, delete the `rods` dictionary and turn its keys into the variables `A`, `B`, and `C`, respectively, keeping their values. Refactor your code to reflect these changes. If you see the output on the terminal, you correctly did this step. 
+
+# --hints--
+
+You should turn `rods['A']` into a variable named `A` keeping its value.
+
+```js
+({ test: () => assert.match(code, /A\s*=\s*list\s*\(\s*range\s*\(\s*NUMBER_OF_DISKS\s*,\s*0\s*,\s*-\s*1\s*\)\s*\)/) })
```

Comment:
It is confusing to have done this, but because the code does not "compile", this is the hint that gets given.

```suggestion
assert.match(code, /A\s*=\s*list\s*\(\s*range\s*\(\s*NUMBER_OF_DISKS\s*,\s*0\s*,\s*-\s*1\s*\)\s*\)/);
```